app/renderer/api/getWallet.ts

```diff
@@ -17,6 +18,13 @@ export async function getWallet(client: ApiClient, id: string, password: string)
   : Promise<GetWalletResponse> {
   return client.request("getWallet", { id, password })
     .then(parseResponse)
+    .then((response) => {
```

Comment:
As this PR includes commits from this other [PR](https://github.com/witnet/sheikah/pull/359), maybe you could apply the comments made in there (or coordinate with @anler to see who implements them)

Reply:
I implemented some of the changes of [PR](https://github.com/witnet/sheikah/pull/359) the other changes are included in the [PR](https://github.com/witnet/sheikah/pull/372) of the issue #333